src/main/java/pl/javastart/task/Main.java

```diff
@@ -1,16 +1,94 @@
 package pl.javastart.task;
 
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
 import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 
 public class Main {
+    private static final Pattern[] patterns = new Pattern[]{
+            Pattern.compile("(\\d{4})-(0[1-9]|1[0-2])-(\\d{2}) (\\d{2}):(\\d{2}):(\\d{2})"),
+            Pattern.compile("(\\d{2}).(0[1-9]|1[0-2]).(\\d{4}) (\\d{2}):(\\d{2}):(\\d{2})"),
```

Comment:
Ten format nie jest prawidłowy, kropka to dowolny znak, więc np. pasuje tutaj data w formacie `05,12,2015 22:00:00`
Powinnaś wyeskejpować kropkę `\\.`